tests/test_search.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+import requests
+from unittest.mock import patch, Mock
+from morphosource.search import search_media, get_media, Endpoints, ItemNotFound
+from morphosource.download import DownloadVisibility
+
+MS_MEDIA = [
+    {
+        'id': ['000390223'],
+        'title': ['Dentary Teeth [Mesh] [CT]'],
+        'media_type': ['Mesh'],
+        "visibility": ["Open Download"],
+    },
+    {'id': ['000390218'], 'title': ['Maxillary Teeth [Mesh] [CT]'], 'media_type': ['Mesh']},
```

Comment:
```suggestion
    {
        'id': ['000390218'], 
        'title': ['Maxillary Teeth [Mesh] [CT]'], 
        'media_type': ['Mesh']
    },
```
Formatting consistency makes for easier reading. I assume it's list of three items vs four, but it doesn't register as matching. Do they have entries that don't list visibility?